.eslintrc.js

```diff
@@ -10,10 +6,6 @@ module.exports = {
   },
   parser: '@typescript-eslint/parser', // Specifies the ESLint parser
   extends: [
-    'airbnb-base',
-    'plugin:import/errors',
-    'plugin:import/warnings',
-    'plugin:import/typescript',
```

Comment:
I removed the `import` plugin as it didn't provide a lot of value anyway, and it didn't support PnP out of the box (maybe it does with some config?). I removed `airbnb-base` because it depends on `import` and I also felt like it wasn't super helpful.